src/cephadm/cephadmlib/container_types.py

```diff
@@ -665,14 +665,10 @@ def enable_shared_namespaces(
 
 def get_mgr_images() -> dict:
     """Return dict of default mgr images"""
-    mgr_prefix = 'mgr/cephadm/container_image_'
     mgr_images = {}
-    images = vars(default_images)
-    for key, value in images.items():
-        if key.startswith('DEFAULT_') and key.endswith('_IMAGE'):
-            # flake8 and black disagree about spaces around ":" hence the noqa comment
-            suffix = key[
-                len('DEFAULT_') : -len('_IMAGE')  # noqa: E203
-            ].lower()
-            mgr_images[mgr_prefix + suffix] = value
+    mgr_prefix = 'mgr/cephadm/'
+    for image in DefaultImages:
+        mgr_images.update(
+            {f'{mgr_prefix}{image.value.key}': image.value.name}
+        )
```

Comment:
This is equivalent to:

mgr_prefix = 'mgr/cephadm/'
mgr_images = {f"{mgr_prefix}{image.value.key}": image.value.name for image in DefaultImages}